packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.js

```diff
@@ -7,10 +7,29 @@ import { LightningElement, api } from 'lwc';
  * <c-quantic-card-container title="Card Example"></c-quantic-card-container>
  */
 export default class QuanticCardContainer extends LightningElement {
-    /**
-     * The title label to display in the card header.
-     * @api
-     * @type {string}
-     */
-    @api title;
-}
\ No newline at end of file
+  /**
+   * The title label to display in the card header.
+   * @api
+   * @type {string}
+   */
+  @api title;
+  /**
+   * The title label to display in the card header.
+   * @api
+   * @type {VoidFunction}
+   */
+  @api toggleVisibility;
+
+  /**
+   * @param {KeyboardEvent} evt
+   */
+  onKeyDown(evt) {
+    if (
+      (evt.code === 'Enter' || evt.code === 'Space') &&
+      this.toggleVisibility
+    ) {
+      evt.preventDefault();
+      this.toggleVisibility();
+    }
+  }
```

Comment:
```suggestion
  handleHeaderClick(evt) {
     const headerClickEvent = new CustomEvent('headerclick', {...evt});
     this.dispatchEvent(headerClickEvent);
  }
  
  handleHeaderKeyDown(evt) {
     const headerKeyDownEvent = new CustomEvent('headerkeydown', {...evt});
     this.dispatchEvent(headerKeyDownEvent);
  }
```

Reply:
What do you think of this?

Reply:
This implementation is awesome, added Implementation.